docs/en/installation/integrating.md

```diff
@@ -170,3 +170,13 @@ build using the `tk-doc-generator`:
 [![Doc Generator](https://img.shields.io/badge/Built%20With-SG%20Doc%20Generator-blue.svg)](http://github.com/shotgunsoftware/tk-doc-generator)
 ```
 
+## Configuring Search
+
+The documentation system supports integration with [Google Custom Search](https://cse.google.com). This makes it easy to integrate site specific search results. This is disabled by default - in order to enable it, set up a google custom search profile and then add its search engine id into the configuration:
+
+```yaml
+# add a google custom search box
+# (see https://developers.google.com/custom-search/docs/element)
+google_custom_search_id: 0156836942234623426942:2l54_vxygup
```

Comment:
Do we want to use something like `your-key-here` in the example code?  I know we're not concerned with leaking these keys, but it might make copy/paste errors (due to forgetting to change the code) less likely.

Reply:
good point! (this key is not real btw, but i agree it's better practice to be really explicit about it!)